src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  /* stylelint-disable-next-line string-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 500;
+
+}
+
+body {
+  margin: 0;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header_logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  background-color: #fff;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
```

Comment:
you can use :not(:last-child) instead of :last-child
```
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

Reply:
@Viktor-Kost if I change to your option, the margin tests fail

Reply:
@Viktor-Kost sorry, I changed it in the wrong place. everything works